app-games/src/main/java/com/aptoide/android/aptoidegames/feature_payments/presentation/PaymentButtonsView.kt

```diff
@@ -0,0 +1,87 @@
+package com.aptoide.android.aptoidegames.feature_payments.presentation
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import cm.aptoide.pt.extensions.PreviewDark
+import com.aptoide.android.aptoidegames.R
+import com.aptoide.android.aptoidegames.R.string
+import com.aptoide.android.aptoidegames.design_system.PrimaryButton
+import com.aptoide.android.aptoidegames.theme.AGTypography
+import com.aptoide.android.aptoidegames.theme.AptoideTheme
+import com.aptoide.android.aptoidegames.theme.Palette
+import kotlin.random.Random
+
+@PreviewDark
+@Composable
+fun PreviewPaymentButtons() {
+  val onBuyClickEnabled = Random.nextBoolean()
+  AptoideTheme {
+    Column {
+      PaymentButtons(
+        onBuyClickEnabled = onBuyClickEnabled,
+        onBuyClick = { },
+        onOtherPaymentMethodsClick = { }
+      )
+    }
+  }
+}
+
+@Composable
+fun PaymentButtons(
+  modifier: Modifier = Modifier,
+  onBuyClickEnabled: Boolean = true,
+  onBuyClick: () -> Unit,
+  onOtherPaymentMethodsClick: () -> Unit,
+) {
+  Column(
+    modifier = modifier,
+  ) {
+    DTBuyButton(
+      onBuyClick = onBuyClick,
+      onBuyClickEnabled = onBuyClickEnabled,
+    )
+    DTOtherPaymentMethodsButton(
+      onOtherPaymentMethodsClick = onOtherPaymentMethodsClick
+    )
+  }
+}
+
+@Composable
+fun DTBuyButton(
+  onBuyClickEnabled: Boolean = true,
+  onBuyClick: () -> Unit,
+) {
+  PrimaryButton(
+    title = stringResource(id = string.buy_button),
+    onClick = onBuyClick,
+    enabled = onBuyClickEnabled,
+    modifier = Modifier.fillMaxWidth()
+  )
+}
+
+@Composable
+fun DTOtherPaymentMethodsButton(onOtherPaymentMethodsClick: () -> Unit) {
+  TextButton(
```

Comment:
We need this button in the design system to reuse

Reply:
The text button `AppGamesOtherPaymentMethodsButton` is used instead of this. Should it be in the design system?

Reply:
Of course it should 😄